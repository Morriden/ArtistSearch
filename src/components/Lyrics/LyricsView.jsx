import React from 'react';
import PropTypes from 'prop-types';

const LyricsView = ({ lyrics }) => {
  return (
    <div>
      {lyrics}
    </div>
  );
};

LyricsView.propTypes = {
  lyrics: PropTypes.string.isRequired
};

export default LyricsView;
