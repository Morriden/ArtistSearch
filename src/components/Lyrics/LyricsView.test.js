import React from 'react';
import { shallow } from 'enzyme';
import LyricsView from './LyricsView';

describe('Get Lyrics for a song', () => {
  it('should get the lyrics for a song', () => {
    const wrapper = shallow(<LyricsView lyrics={'check my out im some lyricss'}/>);
    
    expect(wrapper).toMatchSnapshot();
  });
});
