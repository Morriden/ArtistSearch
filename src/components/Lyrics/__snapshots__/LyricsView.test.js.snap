// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Get Lyrics for a song should get the lyrics for a song 1`] = `
<div>
  check my out im some lyricss
</div>
`;
